lib/chef-dk/command/generator_commands.rb

```diff
@@ -69,6 +69,12 @@ module SharedGeneratorOptions
         :description => "Use to set arbitrary attribute KEY to VALUE in the code_generator cookbook",
         :default => [],
         :proc => Proc.new { |s| generator_args <<  s.split("=").map(&:strip) }
+
+      option :branch,
```

Comment:
That makes a bunch of tests fail and I don't know why! Can you or someone help me out today? I'm too unfamiliar with chefdk and ruby to be able to be really effective here.